common_database/build.gradle

```diff
@@ -1,6 +1,13 @@
+buildscript {
+    dependencies {
+        // to fix an issue exposed by spotbugs 2.0.0: https://github.com/spotbugs/spotbugs-gradle-plugin/issues/120
+        classpath "com.google.guava:guava:${pluginGuavaVersion}"
+    }
+}
+
 plugins {
     id "org.owasp.dependencycheck" version "4.0.0.1"
-    id "com.github.spotbugs" version "1.6.5"
+    id "com.github.spotbugs" version "2.0.0"
```

Comment:
I couldn't use a variable for this from the parent project, not entirely sure why